Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the x dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogX { get; set; }
```

Comment:
I would be hesitant to suppose this given #164, though I do think that plotting a linearly defined HeatMap on a log plot is unlikely to occur (it is something this commit makes no effort to address, and isn't compatible, so to speak, with the method of using Screen Space coordinates to place the rectangles). If this would be deemed desirable, then I'd gladly look into it, though I can't at this time think how we would detect whether an axis is logarithmic or not, short of checking the type and inferring the mode from the Translate/InverseTranslate methods, neither of which I think very nice, though the I see that the later is employed by the HeatMap image generation to determine whether the X or Y axes need to be reversed.